### PR TITLE
Added clear messages by user in X history. 

### LINKED ISSUE #195: Clear feature improvements
As requested by Hucota, it would be nice if we were able to use `[]clear <username> <message count>` to remove the last x messages of a specific user. 
Maybe even `[]clear <username> <time>` to remove all messages from a user in the last x amount of time?

### PR DESCRIPTION
**Pull request by AquaXV**
Closes #195

### CODE REVIEW

File: src/Commands/Modules/Moderation/ClearCommand.cs [hunk 1]
```diff
@@ -17,7 +18,17 @@ public partial class ModerationModule : BrackeysBotModule
         public async Task ClearMessagesAsync(
             [Summary("The amount of messages to clear")] int count)
         {
-            var messages = await Context.Channel.GetMessagesAsync(count + 1).FlattenAsync();
+            int maxHistory = Context.Configuration.ClearMessageMaxHistory;
+            if (count + 1 > maxHistory)
```
Comment (YilianSource) [inline src/Commands/Modules/Moderation/ClearCommand.cs]: We can shorten to
```cs
count = count + 1 > maxHistory ? maxHistory : count + 1;
```

Comment (AquaXV) [reply src/Commands/Modules/Moderation/ClearCommand.cs]: Eeeh yea. I was not entirely happy with readability. But I still am not. Stupid newlines for braces {}

Comment (YilianSource) [reply src/Commands/Modules/Moderation/ClearCommand.cs]: If you don't like the ternary method you can always just leave out the braces for single-lined if-conditions, no need for newlined braces. :)

Comment (AquaXV) [reply src/Commands/Modules/Moderation/ClearCommand.cs]: Habit from work where we are required to do braces at all times

**Pull request merged**